live-hls.js

```diff
@@ -33,6 +35,9 @@ var live;
 var filterPlaylist;
 var getStream;
 var resetStream;
```

Comment:
Although it is within the defaults object, you are using resetStream and stopStream as functions outside of this object. Might want to reconsider the naming?